### DISCUSSION TITLE
Path Loss common values

### DISCUSSION THREAD

**Discussion by mas-oliveira** [srsran/srsRAN_Project#540] [General Help] [answered]
Hello,
I am collecting KPMs using the ZMQ based device provided by srsRAN but I want to adjust the value of the path loss, currently I'm generating values of path loss using a gaussian distribution, but I would like to do it more precisely. Does someone who is running a solution similar to the ZMQ based devices can provide me a path loss data collection? 
Thamks

**Comment by pgawlowicz** [comment]
Hi @mas-oliveira, not sure if I understand your question correctly. You just need to look for some path loss models, generate a path loss trace, and apply the values to the channel. Or do you need sth else?

**Reply by pgawlowicz** [reply]
@mas-oliveira any update on this?

**Reply by mas-oliveira** [reply, marked as answer]
Hello Piotr, I'm now working around with the noise source.
Thanks
